### DISCUSSION TITLE
There was an error during Vitest startup. Check the output for more details.

### DISCUSSION THREAD

**Discussion by oldTimer98** [vitest-dev/vscode#377] [Feedback]
When I open a test project, I get an error message

The problem is as follows：

[INFO 16:46:07] [API] Running Vitest: v1.6.0 (vite.config.ts)
[INFO 16:46:07] [API] Starting Vitest process with Node.js: D:\Volta\node.EXE
[Worker] node:child_process:172
  p.open(fd);
    ^

Error: EBADF: bad file descriptor, uv_pipe_open
    at Object._forkChild (node:child_process:172:5)
    at setupChildProcessIpcChannel (node:internal/bootstrap/pre_execution:407:30)
    at prepareMainThreadExecution (node:internal/bootstrap/pre_execution:67:3)
    at node:internal/main/run_main_module:7:1 {
  errno: -4083,
  code: 'EBADF',

**Comment by oldTimer98** [comment]
![image](https://github.com/vitest-dev/vscode/assets/66023496/0d27a5e5-79ad-4248-a7d7-92651b5ba98d)

**Comment by kaisalmen** [comment]
@oldTimer98 I have the same issue with the vitest extension with volta\node on Windows. This problem does not occur on Linux.
As a workaround you can enforce another node executable in the preferences (e.g. `"vitest.nodeExecutable": "C:\\Program Files\\nodejs\\node.exe"`)
